source/developer/interactive-message-buttons.rst

```diff
@@ -1,209 +1,4 @@
-
-The following payload gives an example webhook that uses message buttons.
-
-.. code-block:: text
-
-  {
-    "attachments": [
-      {
-        "pretext": "This is the attachment pretext.",
-        "text": "This is the attachment text.",
-        "actions": [
-          {
-            "name": "Ephemeral Message",
-            "integration": {
-              "url": "http://127.0.0.1:7357",
-              "context": {
-                "action": "do_something_ephemeral"
-              }
-            }
-          }, {
-            "name": "Update",
-            "integration": {
-              "url": "http://127.0.0.1:7357",
-              "context": {
-                "action": "do_something_update"
-              }
-            }
-          }
-        ]
-      }
-    ]
-  }
-
-The integration can respond with an update to the original post, or with an ephemeral message:
-
-.. code-block:: text
-
-  {
-    "update": {
-      "message": "Updated!"
-    },
-    "ephemeral_text": "You updated the post!"
-  }
-
-.. image:: ../../source/images/interactive_message.gif
-
-Below we give a brief description of each parameter to help you customize the webhook post in Mattermost. For more information on message attachments, `see our documentation <https://docs.mattermost.com/developer/message-attachments.html>`_.
-
-Name
-  Give your action a descriptive name.
-
-URL
-  The actions are backed by an integration that handles HTTP POST requests when users click the message button. The URL parameter determines where this action is sent to. The request contains an ``application/json`` JSON string.
-
-Context
-  The requests sent to the specified URL contain the user id, post id, channel id, team id, and any context that was provided in the action definition. The post id can be used to, for example, delete or edit the post after clicking on a message button.
-  
-  A simple example of a request is given below:
-  
-  .. code-block:: text
-
-    {
-    "user_id": "rd49ehbqyjytddasoownkuqrxe",
-    "post_id": "gqrnh3675jfxzftnjyjfe4udeh",
-    "channel_id": "j6j53p28k6urx15fpcgsr20psq",
-    "team_id": "5xxzt146eax4tul69409opqjlf",
-    "context": {
-      "action": "do_something"
-      }
-    }
-
-  In most cases, your integration will do one or both of these things:
-  
-  1. **Identifying which action was triggered**. For example, a GitHub integration might store something like this in the context:
-
-    .. code-block:: text
-
-      {
-      "user_id": "rd49ehbqyjytddasoownkuqrxe",
-      "post_id": "gqrnh3675jfxzftnjyjfe4udeh",
-      "channel_id": "j6j53p28k6urx15fpcgsr20psq",
-      "team_id": "5xxzt146eax4tul69409opqjlf",
-      "context": {
-        "repo": "mattermost/mattermost-server"
-        "pr": 1234,
-        "action": "merge"
-        }
-      }
-      
-  When the message button is clicked, your integration sends a request to the specified URL with the intention to merge the pull request identified by the context.
```

Comment:
Consider revising this sentence, "In the example above, when the message button it clicked...."